crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -473,3 +477,41 @@ pub struct IatapayAccessTokenErrorResponse {
     pub error: String,
     pub path: String,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct IatapayPaymentWebhookBody {
+    pub status: IatapayWebhookStatus,
```

Comment:
incase of payment failure error_message, error_code and error_reason should be captured

Reply:
we are already doing this for payments!